vault/request_handling.go

```diff
@@ -141,7 +141,9 @@ func (c *Core) fetchACLTokenEntryAndEntity(ctx context.Context, req *logical.Req
 		var err error
 		te, err = c.tokenStore.Lookup(ctx, req.ClientToken)
 		if err != nil {
-			c.logger.Error("failed to lookup token", "error", err)
+			if !c.IsDRSecondary() {
```

Comment:
Same question here -- I'm worried about using `!c.IsDRSecondary() ` too generously when we're deep in call chains. For example here, if we're sure that DR Secondaries would return an error during a lookup token, I think we should try to avoid the call to the `Lookup` call chain altogether.

Reply:
based on [this](https://hashicorp.atlassian.net/browse/VAULT-2257) it seems at least one of these two paths is being hit on the DR Secondary, resulting in the error

maybe the calls could be avoided in the first place

Reply:
i did a bunch of testing and didn't see anything that hit this path. the ticket error implies one of the two is being hit, but i haven't seen it so far. so at least if it happens on new versions we'll have a better idea what's going on